templates/certified/index.html

```diff
@@ -6,250 +6,632 @@
+      <p>CPU</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-sv2">
+    <hr class="is-fixed-width">
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h2>Regular testing<br />for up to 10 years</h2>
+      <p>Roughly every 3 weeks, Ubuntu releases <a href="https://wiki.ubuntu.com/StableReleaseUpdates">Stable Release Updates</a>, ensuring a secure and reliable experience. These updates are carefully tested by the Hardware Certifications team to make sure that systems work well with Ubuntu.</p>
+      <p><a href="/about/release-cycle">Learn more about Ubuntu release lifecycle</a></p>
+    </div>
 
+    <div class="col-6 u-hide--medium u-hide--small u-align--center">
+      <div class="p-strip--light">
+        <div class="u-fixed-width">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/aa1dc625-10+Year+shield+trio_AW.svg",
+            alt="",
+            width="293",
+            height="161",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
```

Comment:
"People and processes behind certification" section in the copy doc is missing. Is it intended?